contrib/download-frozen-image-v2.sh

```diff
@@ -44,6 +44,18 @@ if [ "$(go env GOHOSTOS)" = 'windows' ]; then
 	fi
 fi
 
+fetch_blob() {
+	url=$1
+	token=$2
+	dest=$3
+	echo "Attempting to download blob $url"
+	target=$(curl -v -sS -I -H "Authorization: Bearer $token" "$url" 2>&1 | grep "Location:" | head -n 1 | sed "s/< Location: \([:graph:]*\)/\1/")
+	printf "\n\nFound redirect target: '$target'\n\n"
+	# curl blob (exclude auth token)
```

Comment:
Ah, looks to be a known issue with cURL; https://stackoverflow.com/questions/37865875/stopping-curl-from-sending-authorization-header-on-302-redirect https://stackoverflow.com/questions/37246261/curl-follow-redirect-without-headers-from-original-request

(Wonder if someone opened an issue with them)